src/components/userComponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,557 @@
+                </svg>
+              </Link>
+            </div>
+          </div>
+        </div>
+        <hr className="border-silver w-[322px] m-auto h-[.5px] mt-5 mb-8" />
+        <div className="bg-scary-forest flex justify-around text-center text-ivory font-[Lato] gap-4 active:text-[#bcc0bb]">
+          <div className="flex flex-col gap-4">
+            <Link href="/hoursAdmissionPage">
+              {' '}
+              <p> Hours & Locations </p>{' '}
+            </Link>
+            <Link href="/siteMapPage">
+              {' '}
+              <p> Site Maps </p>{' '}
+            </Link>
+            <Link href="/featuredToursPage">
+              {' '}
+              <p> Featured Tours </p>{' '}
+            </Link>
+          </div>
+          <div className="flex flex-col gap-4">
+            <Link href="/exhibitsPage">
+              {' '}
+              <p> Exhibits </p>{' '}
+            </Link>
+            <Link href="/newsFeedPage">
+              {' '}
+              <p> News </p>{' '}
+            </Link>
+            <Link href="/spotlightPage">
+              {' '}
+              <p> Wildlife Spotlights </p>{' '}
+            </Link>
+          </div>
+        </div>
+        <div className="w-full pt-6 gap-4 bg-scary-forest">
+          <Link href="https://phs-spca.org">
+            <div className="w-[200px] m-auto px-4 py-1.5 bg-hunterGreen rounded-lg">
+              <div className="text-center text-silver text-xs font-normal font-['Lato'] active:text-[#bcc0bb]">
+                Learn more about PHS/SPCA
+              </div>
+            </div>
+            <div className="mt-4 text-zinc-500 text-center text-xs font-normal font-['Lato']">
+              Copyright © 2024 Peninsula Humane Society & SPCA{' '}
+            </div>
+          </Link>
+          <Link
+            href="https://phs-spca.org/privacy-policy/"
+            className="mt-1 pb-10 text-center flex justify-center text-zinc-500 text-xs font-normal font-['Lato'] underline"
+          >
+            Privacy Policy
+          </Link>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+/**
+ *
+ * @param root0 web component
+ * @param root0.inputValueName name
+ * @param root0.inputValueEmail email
+ * @param root0.subscribed whether you subscribed
+ * @param root0.handleNameChange change name func
+ * @param root0.handleEmailChange change email func
+ * @param root0.handleSubmit what to do when submit email name
+ * @param root0.showError error
+ * @param root0.errorMsg error
+ * @returns web footer
+ */
+function WebInput({
+  inputValueName,
+  inputValueEmail,
+  subscribed,
+  handleNameChange,
+  handleEmailChange,
+  handleSubmit,
+  showError,
+  errorMsg,
+}: EmailInputProps) {
+  return (
+    <div className="flex flex-col bg-scary-forest w-full">
+      <div className="flex pt-24 justify-between">
+        <div className="ml-56 flex flex-col w-[255px] h-[215px] gap-4">
```

Comment:
rem or tailwind units.  It is likely you can change margins to padding but more of a nit